test/helpers/fixture_set.rb

```diff
@@ -28,7 +28,14 @@ def app_name
     end
 
     def refute_resource_exists(type, name, beta: false)
-      client = beta ? v1beta1_kubeclient : kubeclient
+      client = if beta
+        v1beta1_kubeclient
+      elsif %w(daemonset deployment replicaset statefulset).include?(type)
+        apps_v1_kubeclient
+      else
+        kubeclient
+      end
```

Comment:
Do we need to worry about the v1beta1 client? From the docs, it appears that apps/v1 should work in this case, no? Might be worth trying out if you haven't already

Reply:
Yup, we need to care because `Ingress` kind still uses it 😢